base/set.jl

```diff
@@ -191,6 +191,37 @@ function unique(f::Callable, C)
 end
 
 """
+	unique!(itr)
+
+Remove duplicate items and return the modified collection,
+as determined by [`isequal`](@ref).
+
+```jldoctest
+julia> unique!([1, 1, 1])
+1-element Array{Int64,1}:
+ 1
+
+julia>  unique!([7, 3, 2, 3, 7, 5])
```

Comment:
Really pedantic, but there's an extra space here between `julia>` and `unique!`.